### PR TITLE
Add amount to previous expense

### PR DESCRIPTION
**Pull request by rotimi-babalola**
- Adds a feature where a 'new' expense with an existing name adds the amount to the already existing amount in the DB. 

Not sure if I'm explaining this correctly 😅

### CODE REVIEW

File: server/schemas/mutations/add-expense.mutation.js [hunk 1]
```diff
@@ -25,11 +25,11 @@ export default {
   },
   resolve: async (_, { input }) => {
     try {
-      const newExpense = await new Expense(input);
-      if (newExpense) {
-        return newExpense.save();
+      const newExpense = await Expense.findOneOrCreate(input);
+      if (newExpense.isOldExpense) {
```
Comment (ghost) [inline server/schemas/mutations/add-expense.mutation.js:29]: :awesome:

File: server/schemas/mutations/add-expense.mutation.js [hunk 2]
```diff
@@ -25,11 +25,11 @@ export default {
   },
   resolve: async (_, { input }) => {
```
Comment (ghost) [inline server/schemas/mutations/add-expense.mutation.js:26]: By the way, been thinking of asking about the `_`, but I always forget. What does it do if I may ask?

**Review by ghost — COMMENTED** [review]
.

File: server/schemas/mutations/add-expense.mutation.js (hunk 2 above)
Comment (rotimi-babalola) [reply server/schemas/mutations/add-expense.mutation.js:26]: It's just a way of ignoring the first argument to the function @oreoluwade

**Comment by rotimi-babalola** [comment]
What is the meaning of . ? 🤔

**Review by ghost — APPROVED** [review]

**Pull request merged**